pkg/types/v1/common.go

```diff
@@ -35,31 +35,31 @@ const (
 
 // ImageSource represents the source from where an image is created for easy identification
 type ImageSource struct {
-	source  string
-	srcType string
+	Source  string
+	SrcType string
```

Comment:
These need to be exported for `yip` to fill them in when unmarshalling

Reply:
Do they? They don't get unmarshalled via viper directly no? Don't they get filled by the newsrcfromuri method? Maybe we can get away without exporting them?


Not that it matters much to be honest, just trying to understand how it worked before...

Reply:
You are right. I was trying to set the fields directly in the yaml initially before I realized we have the custom unmarhaller. Reverted.